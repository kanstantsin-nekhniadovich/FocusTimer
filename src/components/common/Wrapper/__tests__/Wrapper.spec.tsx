import React from 'react';
import { render } from '@testing-library/react-native';

import { Wrapper } from '..';

describe('<Wrapper />', () => {
  it('show match snapshot', () => {
    const tree = render(<Wrapper />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
