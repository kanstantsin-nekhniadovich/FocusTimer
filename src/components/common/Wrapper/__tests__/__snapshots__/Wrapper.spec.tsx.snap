// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wrapper /> show match snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fbfbfb",
      "flex": 1,
      "position": "relative",
    }
  }
/>
`;
